<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetFlyTask</class>
 <widget class="QDialog" name="SetFlyTask">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>647</width>
    <height>399</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>340</y>
     <width>141</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Полетная задача:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>10</y>
     <width>251</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Параметры двигателя:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>350</y>
     <width>211</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>*все параметры вводятся в системе си</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_warning">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>255</y>
     <width>341</width>
     <height>61</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>170</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>170</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="96">
         <red>164</red>
         <green>166</green>
         <blue>168</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="text">
    <string>err</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>190</width>
     <height>278</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Максимальная высота полета цели</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="le_Hmax"/>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Минимальная высота полета ракеты</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="le_Hmin"/>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Дальняя граница зоны поражения</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="le_Xmax"/>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Рубеж выполнения задания</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="le_mstone"/>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Скорость полета цели</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="le_Vtar"/>
    </item>
    <item>
     <widget class="QLabel" name="label_16">
      <property name="text">
       <string>Угол накл. траектории к ЛВЗ(рад)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="le_teth">
      <property name="text">
       <string>0.0873</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>60</y>
     <width>158</width>
     <height>141</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>Смесь заряда ТТ</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cb_fuel"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_14">
        <property name="text">
         <string>Давление в кр. сечении сопла</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="le_Pk">
        <property name="text">
         <string>10</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_15">
        <property name="text">
         <string>Давление на срезе сопла</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="le_Pa">
        <property name="text">
         <string>0.08</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>61</y>
     <width>168</width>
     <height>188</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_7">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Материал корпуса</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cb_cyl"/>
      </item>
      <item>
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Материал ТЗП</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cb_tzp"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Материал бронирования заряда</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cb_br"/>
      </item>
      <item>
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>Материал корпуса сопла</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cb_nozzle"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SetFlyTask</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
